### DISCUSSION TITLE
I can't interact with the downloaded file using the Facet API as with a TemporaryFile

### DISCUSSION THREAD

**Discussion by FyZzyss** [fastapi/fastapi#7271] [Questions]
I am downloading a file using FastAPI as shown in the documentation. I want to work with it as a `TemporaryFile`. But with the help of `os.path.exists()` I see that this file no. How can I fix this?

My code:

    @router.post(
        path="/po/{id_po}/upload",
        response_model=schema.ContentUploadedResponse,
    )
    async def upload_file(
            id_po: int,
            uploaded_file: UploadFile = File(...)):
        """Загружает файл во временной хранилище"""
        content = await uploaded_file.read()
        uploaded_file.file.rollover()
        print(uploaded_file.file.name)
        print(os.path.abspath(str(uploaded_file.file.name)))
        if content.__sizeof__() < 2097152:
            # await download_user_file(paths=[os.path.abspath(str(uploaded_file.file.name))], user_id=id_po)
            print(os.path.exists(os.path.abspath(str(uploaded_file.file.name))))
            return schema.ContentUploadedResponse(filename=f'{id_po}__{uploaded_file.filename}')
        else:
            raise FileIsTooBig(message='File is bigger than 2 MB')

**Comment by tiangolo** [comment]
I assume you solved it, thanks for closing the issue :+1:
